src/game.py

```diff
@@ -126,17 +142,30 @@ def load_from_json(json_dict, source='lichess'):
 
         return new_game
 
-    def get_result(self):
+    def get_result(self, user):
+
         result = ""
-        if ((self.players['white'].name == "carequinha" and self.result == "1-0") or
-                (self.players['black'].name == "carequinha" and self.result == "0-1")):
+        if user not in self.players.values():
+            result = "Error - that player is not in this game."
+            warnings.warn("That player is not in this game", Warning)
+        elif ((self.players['white'].name == user and self.result == "1-0") or
+              (self.players['black'].name == user and self.result == "0-1")):
             result = "WIN"
-        elif ((self.players['white'].name == "carequinha" and self.result == "0-1") or
-                (self.players['black'].name == "carequinha" and self.result == "1-0")):
+        elif ((self.players['white'].name == user and self.result == "0-1") or
+              (self.players['black'].name == user and self.result == "1-0")):
             result = "LOSS"
         elif self.result == "1/2-1/2":
             result = "DRAW"
 
         if result is None:
             raise RuntimeError("Get_result does not fall in any of the conditions.")
         return result
+
+    def get_clocks(self):
+        if isinstance(self.moves, list):
+            if "clk" in self.moves[1]:
+                clocks_strings = list(map(lambda s: s[8:15], self.moves[1::2]))
+                return list(map(
```

Comment:
Maybe list comprehension?

`[compute_delta_time(datetime.strptime(s, '%H:%M:%S')) for s in clocks_strings]`

Reply:
Sorted!